src/murfey/client/contexts/clem.py

```diff
@@ -221,17 +232,21 @@ def post_transfer(
 
             # Post message if all files for the associated series have been collected
             # .get(series_name, 0) returns 0 if no associated key is found
-            if len(self._tiff_series[series_name]) == 0:
+            if len(self._tiff_series.get(series_name, [])) == 0:
```

Comment:
```suggestion
            if not len(self._tiff_series.get(series_name, [])):
```
is a bit more pythonic